azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/utils/AppServiceConfigUtils.java

```diff
@@ -70,38 +72,35 @@ public static AppServiceConfig buildDefaultConfig(String subscriptionId, String
         return appServiceConfig;
     }
 
-    public static void mergeAppServiceConfig(AppServiceConfig config1, AppServiceConfig config2) {
-        if (config1.region() == null) {
-            config1.region(config2.region());
+    public static void mergeAppServiceConfig(AppServiceConfig to, AppServiceConfig from) {
+        try {
+            mergeObjects(to, from);
+        } catch (IllegalAccessException e) {
+            throw new AzureToolkitRuntimeException("Cannot copy object for class AppServiceConfig.", e);
```

Comment:
Shall we handle the `IllegalAccessException` for config and runtime together? Besides, should we throw `AzureToolkitRuntimeException` in this case? As user may have full configuration and empty default config may not block operation in this case

Reply:
We shall always block program execution since this is a fatal error, stop everything and notice us.